code/stable/dblpend/SRS/mdBook/assets.csv

```diff
@@ -0,0 +1,7 @@
+../../../../traceygraphs/dblpend/allvsall.svg,src/assets/allvsall.svg
+../../../../traceygraphs/dblpend/allvsr.svg,src/assets/allvsr.svg
+../../../../traceygraphs/dblpend/avsa.svg,src/assets/avsa.svg
+../../../../traceygraphs/dblpend/avsall.svg,src/assets/avsall.svg
+../../../../traceygraphs/dblpend/refvsref.svg,src/assets/refvsref.svg
```

Comment:
I wonder if this script should be extended to all of our case studies and used similarly. When I open a fresh build of any project, images don't show, so this would be helpful.